### PR TITLE
Python 2/3 compatibility

### PR DESCRIPTION
**Pull request by return42**
Hi @rfk, here is my next PR for Python 2/3 compatibility we discussed in [1].

Please note; as long as requirements not released on PyPi we have to use them from github. Thats why you will find entries in the requirements.txt like:

- git+https://github.com/mozilla-services/tokenlib
- git+https://github.com/mozilla-services/hawkauthlib.git

Thanks!

-- Markus --

[1] mozilla-services/syncserver#97 (comment)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+pyramid >= 1.6
+
+# tokenlib >= 2.0.0
+# use github as long as it is not released on PyPi
+git+https://github.com/mozilla-services/tokenlib
+
+# hawkauthlib >= 2.0.0
+# use github as long as it is not released on PyPi
+git+https://github.com/mozilla-services/hawkauthlib.git
```
Comment (rfk) [inline requirements.txt]: v2.0.0 for both of these should be in PyPI now

**Comment by rfk** [comment]
This looks good to me; let me know when you have a chance to update the deps to published versions, and I'll merge and tag a new release.

**Comment by return42** [comment]
Hi @rfk, thanks for review! .. deps are now taken from PyPi, see last commit.

**Comment by return42** [comment]
Hi @rfk, just a reminder / I do not want to be impatient / maybe you forgot, if nit sorry for the noise

**Comment by rfk** [comment]
@return42 heh, thanks for the reminder, noise always welcome :-)

I have this in my review backlog from the holiday break, will definitely get to it before the end of the week

**Comment by return42** [comment]
@rfk thanks for the merge! I have sen that you have already upload v2.0.0 to PyPi. The version number here in the repo is still v.2.0.0.rc1. Do you forgot to push your local commit with correct version number?

**Comment by rfk** [comment]
> Do you forgot to push your local commit with correct version number?

Indeed I did, thanks for spotting :-)

**Pull request merged**